client/__tests__/e2e/cellxgeneActions.ts

```diff
@@ -405,11 +429,10 @@ export async function checkGenesetDescription(
 
       const editButton = `${genesetName}:edit-genesetName-mode`;
       await page.getByTestId(editButton).click({
-        force: true
+        force: true,
         /**
          * (thuang): Don't wait for the default timeout, since we want to fail fast
-         */,
-        timeout: 1 * 1000,
+         */ timeout: 1 * 1000,
```

Comment:
nit: just curious, is it auto-formatting that put the comment and the code on the same line?
```suggestion
         */ 
         timeout: 1 * 1000,
```